# Common configuration for APM2 autopilot
#
# node:
#startup_px4_usb_quirk: false

# sys_status
conn_heartbeat: 5.0 # send hertbeat every n secs
conn_timeout: 10.0  # hertbeat timeout n secs
sys:
  min_voltage: 10.0  # diagnostics min voltage

# imu_pub
imu:
  frame_id: "fcu"
  # need find actual values
  linear_acceleration_stdev: 0.0003
  angular_velocity_stdev: !degrees 0.02
  orientation_stdev: 1.0
  magnetic_stdev: 0.0

# gps
gps:
  frame_id: "gps"
  #time_ref_source: "gps"

# param
# None

# waypoint
mission:
  pull_after_gcs: true  # update mission if gcs updates

# pc_io
# None

# command
# None

# *_position
#position:
#  # local_position
#  local:
#    send_tf: false
#    frame_id: "local_origin"
#    child_frame_id: "fcu"
#
#  # vision_pose_estimate [extra]
#  vision:
#    pose_with_covariance: false
#    listen_tf: false
#    frame_id: "local_origin"
#    child_frame_id: "vision"
#    tf_rate_limit: 10.0

# *_setpoint
#setpoint:
#  # setpoint_accel
#  accel:
#    send_force: false
#
#  # setpoint_attitude
#  attitude:
#    listen_tf: false
#    listen_twist: false
#    pose_with_covariance: false
#    frame_id: "local_origin"
#    child_frame_id: "attitude"
#    tf_rate_limit: 10.0
#
#  # setpoint_position
#  position:
#    listen_tf: false
#    frame_id: "local_origin"
#    child_frame_id: "setpoint"
#    tf_rate_limit: 10.0
#
#  # setpoint_velocity
#  # None

# safety_area
#safety_area:
#  p1:
#    x: 1.0
#    y: 1.0
#    z: 1.0
#  p2:
#    x: -1.0
#    y: -1.0
#    z: -1.0

# 3dr_radio
# None

# global_position
#global_position:
#  send_tf: false
#  frame_id: "local_origin"
#  child_frame_id: "fcu"
#  rot_covariance: 99999.0

# vfr_hud
# None

# ftp
# None

# vision_speed_estimates [extras]
#vision_speed:
#  listen_twist: false

# px4flow [extras]
#optical_flow_tx: false  # publish mode (rx from FCU)

# image_pub [extras]
# None

# mocap_pose_estimate [extras]
#mocap:
#  # select mocap source
#  use_tf: false   # ~mocap/tf
#  use_pose: true  # ~mocap/pose

# vim: set et fenc=utf-8 ff=unix ft=yaml sts=2 sw=2 ts=2 :
